charts/common/values.yaml

```diff
@@ -12,6 +12,9 @@ strategy:
   ## DaemonSets ignore this
   type: RollingUpdate
 
+# Override the default command for the mainContainer image
```

Comment:
This can simply say

`Override the default args`

command and args are different things

Reply:
Thanks